src/PowerShellEditorServices/Services/DebugAdapter/Debugging/BreakpointApiUtils.cs

```diff
@@ -0,0 +1,145 @@
+                return;
+            }
+
+            s_setLineBreakpointLazy = new Lazy<Func<Debugger, string, int, int, ScriptBlock, LineBreakpoint>>(() =>
+            {
+                MethodInfo setLineBreakpointMethod = typeof(Debugger).GetMethod("SetLineBreakpoint", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Func<Debugger, string, int, int, ScriptBlock, LineBreakpoint>)Delegate.CreateDelegate(
+                    typeof(Func<Debugger, string, int, int, ScriptBlock, LineBreakpoint>),
+                    firstArgument: null,
+                    setLineBreakpointMethod);
+            });
+
+            s_setCommandBreakpointLazy = new Lazy<Func<Debugger, string, ScriptBlock, string, CommandBreakpoint>>(() =>
+            {
+                MethodInfo setCommandBreakpointMethod = typeof(Debugger).GetMethod("SetCommandBreakpoint", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Func<Debugger, string, ScriptBlock, string, CommandBreakpoint>)Delegate.CreateDelegate(
+                    typeof(Func<Debugger, string, ScriptBlock, string, CommandBreakpoint>),
+                    firstArgument: null,
+                    setCommandBreakpointMethod);
+            });
+
+            s_getBreakpointsLazy = new Lazy<Func<Debugger, List<Breakpoint>>>(() =>
+            {
+                MethodInfo removeBreakpointMethod = typeof(Debugger).GetMethod("GetBreakpoints", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Func<Debugger, List<Breakpoint>>)Delegate.CreateDelegate(
+                    typeof(Func<Debugger, List<Breakpoint>>),
+                    firstArgument: null,
+                    removeBreakpointMethod);
+            });
+
+            s_removeBreakpointLazy = new Lazy<Func<Debugger, Breakpoint, bool>>(() =>
+            {
+                MethodInfo removeBreakpointMethod = typeof(Debugger).GetMethod("RemoveBreakpoint", BindingFlags.Public | BindingFlags.Instance);
+
+                return (Func<Debugger, Breakpoint, bool>)Delegate.CreateDelegate(
+                    typeof(Func<Debugger, Breakpoint, bool>),
+                    firstArgument: null,
+                    removeBreakpointMethod);
+            });
+        }
+
+        #endregion
+
+        #region Public Static Properties
+
+        private static Func<Debugger, string, int, int, ScriptBlock, LineBreakpoint> SetLineBreakpointDelegate => s_setLineBreakpointLazy.Value;
+
+        private static Func<Debugger, string, ScriptBlock, string, CommandBreakpoint> SetCommandBreakpointDelegate => s_setCommandBreakpointLazy.Value;
+
+        private static Func<Debugger, List<Breakpoint>> GetBreakpointsDelegate => s_getBreakpointsLazy.Value;
+
+        private static Func<Debugger, Breakpoint, bool> RemoveBreakpointDelegate => s_removeBreakpointLazy.Value;
+
+        private static Func<string, int, int, ScriptBlock, LineBreakpoint> CreateLineBreakpointDelegate => s_newLineBreakpointLazy.Value;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the unused private property 'CreateLineBreakpointDelegate'.](https://app.codacy.com/app/TylerLeonhardt/PowerShellEditorServices/pullRequest?prid=4598462)